src/client/pfs/pfs.proto

```diff
@@ -551,7 +551,7 @@ message CreateTmpFileSetResponse {
 
 message RenewTmpFileSetRequest {
   string fileset_id = 1;
-  int64 ttl_seconds = 2;
+  int64 ttl = 2;
```

Comment:
We should probably add a comment here, explaining the units.

Reply:
Or rename to `ttl_ns`

Reply:
I'm going to convert this back to seconds and use the check you mention below in the client. The field being seconds makes more sense to me after thinking it over a bit.